internal/pkg/cli/app.go

```diff
@@ -37,3 +37,22 @@ An application represents an Amazon ECS service or task.`,
 	}
 	return cmd
 }
+
+// BuildDeployCmd is the deploy command - which is
+// an alias for app deploy.
+func BuildDeployCmd() *cobra.Command {
```

Comment:
nit: should we move this to `deploy.go` and the interfaces in `deploy.go` to `interfaces.go`?

Reply:
yus.